be2-scala/src/main/scala/ch/epfl/pop/model/objects/LaoData.scala

```diff
@@ -9,9 +9,15 @@ import ch.epfl.pop.model.network.method.message.data.lao.CreateLao
 import ch.epfl.pop.model.network.method.message.data.rollCall.CloseRollCall
 import spray.json._
 
+//Ed25519 is used to verify the signatures, therefore I also use it to sign data
+import com.google.crypto.tink.subtle.Ed25519Sign
+
+//a pair of byte arrays works better as keypair than the actual keypair object, since Ed25519Sign.Keypair isn't directly convertible to json with spray
```

Comment:
For readability I would suggest defining a type like this for example.
`type KeyPaire = (Array[Bytes], Array[Bytes])`.
You can then pattern match on the keyPaire(tuple2) or even get public/private key array by using `._1 or ._2`

Reply:
I considered this, however for the practical use, I thought getting the public/private keys with their names instead of ._1 or ._2 (which could lead to confusion) or with getters would be less intuitive than with .publicKey/.privateKey